VERSION.txt

```diff
@@ -1 +1 @@
-5.1.0
\ No newline at end of file
+5.3.0
```

Comment:
This probably only needs to be a patch version bump rather than a minor version bump. You are probably looking at going to 5.1.2 (assuming Dan takes 5.1.1)